src/Upgrader/Configuration/ConfigurationProvider.php

```diff
@@ -42,19 +42,18 @@ public function getIgnoreSources(): array
     }
 
     /**
-     * @return array<string>
+     * @return array<int, string>
      */
     public function getCorePaths(): array
     {
-        $directories = [
-            $this->getVendorPath() . 'spryker' . DIRECTORY_SEPARATOR,
-            $this->getVendorPath() . 'spryker-eco' . DIRECTORY_SEPARATOR,
-            $this->getVendorPath() . 'spryker-shop' . DIRECTORY_SEPARATOR,
-        ];
+        $directories = [];
+        foreach ((array)glob($this->getVendorPath() . 'spryker*', GLOB_ONLYDIR) as $directory) {
```

Comment:
it's not critical, but from my perspective construction with array_filter is easier to read

Reply:
Totally agree with you 
but in this case, we will have PhpStan issue(
![Screenshot from 2022-07-15 13-10-36](https://user-images.githubusercontent.com/85889893/179203149-638dea80-55e3-46ae-a817-d29b15c36b55.png)

Reply:
Fixed